src/istgt_lu.c

```diff
@@ -4507,6 +4510,7 @@ luworker(void *arg)
 			tdiff(second2, third, r);
 #ifdef REPLICATION
 			lu_task->lu_cmd.luworkerindx = tind;
+			clock_gettime(CLOCK_MONOTONIC, &io_queue_time[tind]);
```

Comment:
istgt_lu_disk_queue_start does work of reading from wire also for write IOs.
So, better to move this to the 'replicate' function where IOs will be queued into replicas

Reply:
ok